components/alert_banners/README.md

```diff
@@ -6,57 +6,216 @@ author: orinevares
 * Allow users the ability to dismiss alerts when appropiate.
 * Do not include alerts that are not related to the users current goal.
 
 ## Rationale
-
-## Behaviour
-1. 
+This alert banner uses semantic colours to convey the meaning behind each message. The information is organized with a logo and title at the top to establish the purpose of the alert, while the supporting text immediately below guides the user with further information or supporting actions. 
 
 ## Accessibility
 In addition to [fundamental accessibility]() requirements, this component meets or exceeds the [WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/) standards by incorporating the following criteria.
 
 ### Screenreaders
 As read using ChromeVox
 
-> *""*
+> *"Success. Everything has been submitted. You should expect a response within three business days."*
+
+> *"Error. There are errors in your submission. Please go back and fix them."*
+
+> *"Information. This webpage has been updated to include the latest poluch changes that took effect on September 28, 2018"*
+
+> *"Warning. Effective December 31, 2019 this website will be shutting down permanently."*
+
 
 ### Colour Contrast
-* [Contrast ratio](https://webaim.org/resources/contrastchecker/) exceed 7:1 for text on button when in static and hover states.
+* [Contrast ratio](https://webaim.org/resources/contrastchecker/) exceeds 7:1 for text and links on all alert background colours.
+
+### ARIA Labels
+* ```role="alertdialog"``` labels have been included to ensure screenreaders are conveying the contents of the alert banners effectively.
 
 ## Code
 ### HTML
 ```html
 
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=Noto+Sans" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <title>Alert Banner</title>
+  </head>
+<body>
+
+   <div class="bc-gov-alertbanner-success" role="alertdialog" aria-labelledby="success" aria-describedby="success-desc">
+    <div class="bc-gov-alertbanner-contents">  
+      <i class="fas fa-check-circle"></i>
+      <div class="bc-gov-alertbanner-success-contents-text">
+        <h3 id="success">Success</h3>
+        <p id="success-desc">Everything has been submitted. You should expect a response within three business days.</p>
+      </div>
+    </div>
+    </div>
+
+    <div class="bc-gov-alertbanner-error" role="alertdialog" aria-labelledby="error" aria-describedby="error-desc">
+        <div class="bc-gov-alertbanner-contents">  
+            <i class="fas fa-exclamation-circle"></i>
+          <div>
+            <h3 id="error">Error</h3>
+            <p id="error-desc">There are errors in your submission. Please go back and <a href="">fix them</a>.</p>
```

Comment:
This example doesn't really follow our design guidance since it should tell them exactly what is wrong. The error should also show up on the page they are on, rather taking them to a new page to show the error.